nextcord/application_command.py

```diff
@@ -917,11 +962,10 @@ async def invoke_error(self, interaction: Interaction, error: Exception) -> None
         """|coro|
         Invokes the error handler if available.
         """
-        if self.has_error_handler() and self.error_callback is not None:
-            if self.parent_cog:
-                await self.error_callback(self.parent_cog, interaction, error)
```

Comment:
Why isn't the parent cog supplied as a parameter anymore?

Reply:
Take a look at the [last disscussion with Skelmis](https://github.com/nextcord/nextcord/pull/989#discussion_r1103737943) on this exact same thing.